leetcode/generate_md.py

```diff
@@ -0,0 +1,22 @@
+fin = open('source_leetcode_data.txt', 'r')
```

Comment:
Your Pull Request title: LEETCODE Added generate_md.py

Check result:

* Pull Request title must contain group number in ['1021', '1022', '1013', '2021', '2022']